imagetest/utils/test_utils.go

```diff
@@ -129,3 +129,25 @@ func DownloadPrivateKey(user string) ([]byte, error) {
 	}
 	return privateKey, nil
 }
+
+// GetHostKeysFromDisk read ssh host public key and parse
+func GetHostKeysFromDisk() (map[string]string, error) {
+	bytes, err := ioutil.ReadFile("/etc/ssh/ssh_host_*_key.pub")
+	if err != nil {
+		return nil, err
+	}
+	return ParseHostKey(bytes), nil
+}
+
+// ParseHostKey parse hostkey data from bytes.
+func ParseHostKey(bytes []byte) map[string]string {
```

Comment:
are there any conditions that could be invalid and should return error in this function?

Reply:
I don't find any explicit error returned here.

Reply:
well for example - if any one of these files is badly formatted and has no spaces, this test will panic

Reply:
I add some format check for the hostkey file, it has to be one line per key value and each line has length 2 after split.